.travis.yml

```diff
@@ -23,9 +23,5 @@ script:
   - kill "${pid}"
   - |
     if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
```

Comment:
You can remove this if condition as well. It is covered in `openwisp-utils-qa-checks`

Reply:
correct, thanks @TheOneAboveAllTitan 👍